tests/unit_tests/db/test_db_from_repo_handler.py

```diff
@@ -0,0 +1,123 @@
+        parameters=dict.fromkeys(_pars_telescope_model, None),
+        site="North",
+        telescope_name="MSTN-design",
+        model_version=None,
+        parameter_collection="telescope",
+        db_simulation_model_url=db_config["db_simulation_model_url"],
+        db_simulation_model="verified_model",
+    )
+    assert len(_pars_mstndesign) > 0
+
+    _pars_site_model = [
+        "corsika_observation_level",
+        "geomag_horizontal",
+        "reference_point_utm_east",
+    ]
+
+    _pars_south = db_from_repo_handler._update_parameters_from_repo(
+        parameters=dict.fromkeys(_pars_site_model, None),
+        site="South",
+        telescope_name=None,
+        model_version=None,
+        parameter_collection="site",
+        db_simulation_model_url=db_config["db_simulation_model_url"],
+        db_simulation_model="verified_model",
+    )
+    assert len(_pars_south) > 0
+
+    with caplog.at_level(logging.ERROR):
+        with pytest.raises(ValueError):
+            db_from_repo_handler._update_parameters_from_repo(
+                parameters=dict.fromkeys(_pars_telescope_model, None),
+                site="North",
+                telescope_name="MSTN-01",
+                model_version=None,
+                parameter_collection="not_a_collection",
+                db_simulation_model_url=db_config["db_simulation_model_url"],
+                db_simulation_model="verified_model",
+            )
+        assert "Unknown parameter collection" in caplog.text
+
+    # Test with a parameter that is not in the repository (no error should be raised)
+    _pars_site_model.append("not_a_parameter")
+    db_from_repo_handler._update_parameters_from_repo(
+        parameters=dict.fromkeys(_pars_site_model, None),
+        site="South",
+        telescope_name=None,
+        model_version=None,
+        parameter_collection="site",
+        db_simulation_model_url=db_config["db_simulation_model_url"],
+        db_simulation_model="verified_model",
+    )
+
+
+def test_update_telescope_parameters_from_repo(db_config):
+    _pars_telescope_model = ["telescope_axis_height", "telescope_sphere_radius"]
+    _pars = db_from_repo_handler.update_model_parameters_from_repo(
+        parameters=dict.fromkeys(_pars_telescope_model, None),
+        site="North",
+        telescope_name="MSTN-01",
+        model_version=None,
+        db_simulation_model_url=db_config["db_simulation_model_url"],
+    )
+    assert len(_pars) > 0
+
+
+def test_update_site_parameters_from_repo(db_config):
+    _pars_site_model = [
+        "corsika_observation_level",
+        "geomag_horizontal",
+        "reference_point_utm_east",
+    ]
+    _pars = db_from_repo_handler.update_site_parameters_from_repo(
+        parameters=dict.fromkeys(_pars_site_model, None),
+        site="South",
+        model_version=None,
+        db_simulation_model_url=db_config["db_simulation_model_url"],
+    )
+    assert len(_pars) > 0
```

Comment:
I understand this - but the issue is that we hardcode model values into the tests. I know we do this in other places, but I don't think this testing the functionality (we have otherwise discussions like on the previous item where we wonder why the height changes by 2 m).

In this case we want to be sure that parameters are read from the repository, and a non-zero length test on the return dictionary is covering this sufficiently?

Reply:
it might be that all values are Nan, or 0 if something is broken, but the length would be still correct. we can leave it for another PR when dealing only with the tests.